impl/proc_macros/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "gdnative-impl-proc-macros"
+authors = ["The godot-rust developers"]
+description = "Internal dependency of the gdnative bindings."
+repository = "https://github.com/godot-rust/godot-rust"
+version = "0.1.0"
```

Comment:
This could have the same version as the other crates, so for next release I'd put it at `0.9.0`?

Reply:
`0.9.0` should be fine since that's what we're already at. Changed.